AGENTS.md

```
@@ -0,0 +1,2325 @@
+# ❌ Bad examples
+Add feature                     # Missing type
+feat: Add feature               # Subject not lowercase
+feat!: breaking change          # Exclamation mark not allowed
+fix(Auth): bug fix              # Scope not lowercase
+feat: add feature.              # Period at end
+update docs                     # Invalid type
+```
+
+### Automated Git Hooks with Husky
+
+**Husky automatically enforces code quality on every commit**
```

markdownlint-cli2 (0.18.1)

Emphasis used instead of a heading

(MD036, no-emphasis-as-heading)